Website/index.html

```
@@ -40,7 +51,7 @@ <h1>{{~ listingInfo.Name ~}}</h1>
           <div class="caption1" id="publishedByText">
```

HTMLHint (1.5.0)

[error] The id value [ publishedByText ] must be unique.

(id-unique)